config.example.py

```diff
@@ -1,6 +1,13 @@
-
 money_url = "https://somewhere/x.json"
 money_user = "my_user"
 money_password = "my_basicauth_passwd"
 
 LDAP_HOST = "ldaps://ldap.example.com"
+LDAP_PW = ""
+
+MAIL_FROM = "flipdot-noti@vega.uberspace.de"
```

Comment:
Please add a comment about this mail receiving a lowbalance notification when running in dev...
or remove that functionality ;)